docs/src/examples/ImageExample/ImageExampleFirst.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Image } from '../../../../src/components/';
+import * as styles from '../../styles/components-page.scss';
+
+export interface IProps{
+}
+
+export interface IState {
+}
+
+class ImageExampleFirst extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const url = 'https://www.w3schools.com/css/trolltunga.jpg';
```

Comment:
Should be in state along with alt.

Reply:
I used url and alternate text in property now.so remove 'const' and change accordingly.